.github/workflows/codeql.yml

```diff
@@ -0,0 +1,38 @@
+name: "Code scanning - action"
+
+on:
```

Comment:
can you elaborate on how this will get triggered? i am worried that it will run unnecessarily often.

Reply:
It will be triggered when the pr is submitted, and the inspection results will be reported to the https://sonarcloud.io platform for viewing